src/spikeinterface/core/recording_tools.py

```diff
@@ -149,6 +149,38 @@ def get_closest_channels(recording, channel_ids=None, num_channels=None):
     return np.array(closest_channels_inds), np.array(dists)
 
 
+def get_noise_levels(recording: 'BaseRecording', return_scaled: bool = True, method: Literal['mad', 'std'] = "mad", 
+    force_recompute: bool=False, **random_chunk_kwargs):
+    """
+    Estimate noise for each channel using MAD methods.
+    You can use standard deviation with `method='std'`
+
+    Internally it samples some chunk across segment.
+    And then, it use MAD estimator (more robust than STD)
+
+    """
+    
+    if 'noise_levels' in recording._properties and not force_recompute:
+        noise_levels = recording.get_property(key="noise_levels")
+        return noise_levels
+    else:
```

Comment:
The else nesting here is unecessary after the early return.

Reply:
indeed